templates/cis.rules.erb

```diff
@@ -47,9 +47,15 @@
 -w /var/run/faillock/ -p wa -k logins
 
 # CIS 4.1.9
+<% if node['platform_version'] =~ /^6/ %>
```

Comment:
Are you trying to get RHEL 6 here? This is going to match a whole bunch of platforms if written this way.

Reply:
I based this on the platform version checks for CIS 4.1.4 in the same file. It does need to allow for CentOs and RHEL. If you think this is too generic, I will need to look into refactoring CIS 4.1.4 also.

Reply:
Tim already suggested this, but the PR he made the suggestion on got merged before it could be added:

https://github.com/chef-cookbooks/auditd/pull/43#discussion_r203082747

> A better thing to do here is node['platform_version'].to_i == 6